scopes/dependencies/dependency-resolver/dependency-resolver.graphql.ts

```diff
@@ -43,12 +43,28 @@ export function dependencyResolverSchema(dependencyResolver: DependencyResolverM
         type: String!
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part